src/python/pants/engine/rules_test.py

```diff
@@ -153,19 +155,19 @@ def test_valid_get_unresolvable_subject_declared_type(self) -> None:
             self._parse_rule_gets("Get(int, DNE, 'bob')")
 
     def test_invalid_get_no_subject_args(self) -> None:
```

Comment:
These tests duplicate `selectors_test.py` now. I didn't delete them though because these are small and I'm thinking they act like ~integration tests, i.e. that everything is wired up correctly.